t/porting/bidi-control.t

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/perl
+use v5.20.0;
+use warnings;
+
+use PerlIO::encoding ();
+
+BEGIN {
+  @INC = '..' if -f '../TestInit.pm';
+  require './test.pl';
+}
+
+my %SKIP_FILE = (
+  'cpan/Encode/t/big5-hkscs.enc' => 1, # weird data file and not really source
```

Comment:
Big5 is a non-Unicode encoding.  The hk probably stands for Hong Kong.  I haven't investigated further.